src/WebJobs.Script/Workers/Rpc/WebHostRpcWorkerChannelManager.cs

```diff
@@ -140,6 +140,23 @@ public async Task SpecializeAsync()
             _logger.LogDebug("Completed language worker channel specialization");
         }
 
+        public async Task WorkerWarmup()
+        {
+            _workerRuntime = _environment.GetEnvironmentVariable(RpcWorkerConstants.FunctionWorkerRuntimeSettingName);
+
+            if (_workerRuntime == null)
+            {
+                return;
+            }
+
+            IRpcWorkerChannel rpcWorkerChannel = await GetChannelAsync(_workerRuntime);
+            if (rpcWorkerChannel != null)
+            {
+                _logger.LogDebug("Starting worker warmup");
```

Comment:
nit: same here, extra unnecessary log. we are already logging in SendWorkerWarmupRequest